humilis_sam/meta.yaml.j2

```diff
@@ -103,7 +104,7 @@ meta:
             description:
                 description field of the deployed lambda functions
             value:
-                Function part of a SAM app deployed by {{_env.name}}-{{_env.stage}}
+                Part of a SAM app deployed by {{_env.name}}-{{_env.stage}}
```

Comment:
Since this will appear in the AWS Lambda console, it's redundant to say it's a function.